src/operator/tensor/elemwise_binary_scalar_op_extended.cc

```diff
@@ -74,8 +74,8 @@ where :math:`x` is an element of the tensor *lhs* and :math:`\sigma` is the scal
 
 Example::
 
-  a = mx.nd.array([1, 2, 3, 4]), :math:`\sigma=1`,
-  smooth_l1(a, :math:`\sigma`) = [0.5, 1.5, 2.5, 3.5]
+  a = mx.nd.array([1, 2, 3, 4]), sigma=1,
+  smooth_l1(a, sigma) = [0.5, 1.5, 2.5, 3.5]
```

Comment:
This still looks weird. Its unclear if its code or math.
I think 
```
smooth_l1([1, 2, 3, 4], sigma=1) = [0.5, 1.5, 2.5, 3.5]
```
looks better